v19.2/movr.md

```diff
@@ -0,0 +1,85 @@
+---
+title: MovR
+summary: The MovR application uses CockroachDB to store information about vehicles, users, and rides.
+toc: true
+---
+
+MovR is a fictional vehicle-sharing company created to demonstrate CockroachDB's features.
+
+## Overview
+
+The MovR example consists of three components:
+
+- The `movr` database, which contains several tables that hold vehicle, user, and ride data. The `movr` database is built into `cockroach demo` and `cockroach workload`.
+- The `movr` dataset, which consists of rows of data that populate tables in the `movr` database. The `movr` dataset is built into `cockroach demo` and `cockroach workload`.
+- The MovR application is a fully-functional vehicle-sharing application, written in Python. All of MovR application source code is open-source, and available on the [movr](https://github.com/cockroachdb/movr) GitHub repository.
+
+## The `movr` Database
+
+Three tables in the `movr` database store most of the user, vehicle, and ride data for MovR:
```

Comment:
It may be correct that they store "most of" the data, but perhaps for simplicity of introducing this it could be glossed over?  Genuinely asking because I don't actually know movr that well.  Alternatively, if we do keep it in, IMO we should say where "the rest of it" is.

Reply:
Good call. I removed that "most of" part. There is a sentence further down that lists the other tables (currently) in the database.